webpack.config.js

```diff
@@ -12,33 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const path = require('path');
+
 module.exports = {
-  entry: './src/browser.js',
+  entry: './src/index.ts',
   output: {
-    library: 'vision',
-    filename: './vision.js',
+    library: 'ImageAnnotator',
```

Comment:
let's do `--mainServiceName vision`. Vision is a good name :)

Reply:
got it, configured in `synth.py` and rerun synthtool to make the change. Thank you